admin/client/src/components/FormAction/FormAction.js

```diff
@@ -9,26 +9,33 @@ class FormAction extends SilverStripeComponent {
   }
 
   render() {
-    const props = {
-      type: this.props.type,
-      className: this.getButtonClasses(),
-      disabled: this.props.disabled,
-      onClick: this.handleClick,
-    };
-
-    if (typeof this.props.id !== 'undefined') {
-      props.id = this.props.id;
-    }
-
     return (
-      <button {...props}>
+      <button {...this.getButtonProps()}>
         {this.getLoadingIcon()}
-        {this.props.label}
+        {this.props.title}
       </button>
     );
   }
 
   /**
+   * Get props for the button
+   *
+   * @returns {Object}
+   */
+  getButtonProps() {
+    // Merge attributes
+    return Object.assign({},
+      typeof this.props.attributes === 'undefined' ? {} : this.props.attributes,
```

Comment:
I'd suggestion adding a `propTypes = { attributes:  React.PropTypes.object, }` and `defaultProps = { attributes: {}, }`
So you get cleaner autonomous code in React